src/ml_board/backend/restful_api/data_access.py

```diff
@@ -406,3 +413,34 @@ def delete_checkpoint_resource(
                 os.rmdir(folder_path)
         else:
             raise FileNotFoundError(f"File in path {requested_full_path} not found.")
+
+    def get_system_info(self):
+        """
+        ``HTTP GET`` Fetch System Information for model card.
+
+        :returns: JSON object - System Information of host machine (CPU & GPU)
+        """
+        try:
```

Comment:
this information should not be retrieved upon request. Let's store it the way, as explained above.

Reply:
done